data/json/npcs/refugee_center/surface_staff/Smokes/free_merchant_shopkeep_missions.json

```diff
@@ -0,0 +1,360 @@
+[
+  {
+    "id": "MISSION_FREE_MERCHANTS_HUB_DELIVERY_1",
+    "//": "Breadcrumb quest from Free Merchants -> Hub 01.  Bring a hard drive to Hub 01 from the refugee center and then return.",
+    "type": "mission_definition",
+    "name": { "str": "Unknown Recipient" },
+    "description": "Deliver the FEMA hard drive to the strange group that Smokes mentioned.  Return to the evacuation center when the data's been delivered.",
+    "goal": "MGOAL_CONDITION",
+    "difficulty": 5,
+    "start": {
+      "effect": [ { "u_spawn_item": "fema_data", "count": 1 } ],
+      "assign_mission_target": {
+        "om_terrain": "robofachq_surface_entrance",
+        "om_special": "hub_01",
+        "reveal_radius": 1,
+        "random": true,
+        "search_range": 240
```

Comment:
this will probably be too short of a search range. Just to check - this mission is outside the usual mission chain, correct? It can be very tough if the search doesn't find hub01.

Reply:
It's outside the normal mission chain, yeah. This is actually cut-and-paste from the existing quest def except for the dialogue, which is different - it has a search range of 240 in the current iteration of the game as well. What should I change it to?

Reply:
I went with 400 with the teamster in the back room and we still get "can't find it" errors.

Reply:
Damn. What should I use in the meantime, then? 400, something higher...